fvm/src/kernel/error.rs

```diff
@@ -11,14 +11,14 @@ pub type Result<T> = std::result::Result<T, ExecutionError>;
 /// Convenience macro for generating Actor Errors
 #[macro_export]
 macro_rules! syscall_error {
-    // Error with only one stringable expression
-    ( $code:ident; $msg:expr ) => { $crate::kernel::SyscallError::new(fvm_shared::error::ErrorNumber::$code, $msg) };
```

Comment:
ah so we were always matching this pattern first when passing only a string literal since it taken as expression, that's why you moved it down

Reply:
Yep.